monkey/monkey_island/cc/server_setup.py

```diff
@@ -178,8 +181,20 @@ def _connect_to_mongodb(mongo_db_process: Optional[MongoDbProcess]):
         sys.exit(1)
 
 
-def _start_nextjs_server(data_dir: Path):
-    nextjs_process = start_nextjs(data_dir)
+def _start_nextjs_server(config_options: IslandConfigOptions):
+    if not config_options.node_port:
+        logger.error(
+            f"Node server port is not specified in the config file."
+            f' Specify it as "{NODE_PORT}" in the server configuration file and try again.'
```

Comment:
We should mention where the server configuration file can be found.

Reply:
It's not the responsibility of this method to read the config file, so it doesn't know where the config file is. The method that's responsible for reading the config should log that

Reply:
We should say it's in the data directory.

Reply:
`server_config.json` is not necessarily in data directory. Whoever is responsible for reading it should log it. Also, we'll have docs

Reply:
0857ad7

![image](https://github.com/guardicore/monkey/assets/44770317/e06c36d9-fee1-4e10-a8c8-c802e8f2f347)